src/App.tsx

```diff
@@ -5,16 +5,18 @@ import Login from "./components/auth/Login";
 import { useDispatch, useSelector } from "react-redux";
 import { statesModal } from "./store/todoReducer";
 import Button from "./components/shared/form/Button";
-import { loginTodoAction } from "./store";
+import { setTodoAction, loginTodoAction } from "./store";
+import instance from "./axiosConfig";
 
 function App() {
   const state = useSelector((state: statesModal) => state);
   const dispatch = useDispatch();
-  var accessToken = window.localStorage.getItem("accessToken");
   let formTitle = state.isLogedin ? "Todo List" : "Todo Login";
+  const accessToken = window.localStorage.getItem("accessToken");
```

Comment:
Do we need it here?

Reply:
checked, and was actually no needed, so removed

Reply:
yes, actually it is needed to be used in useEffect, otherwise it is throwing me out on page refresh after login